engineering-feedback/feedback-examples.md

```diff
@@ -0,0 +1,46 @@
+# Engineering Feedback Examples
```

Comment:
My main question here is, who do we expect to be the audience of this document. During our engagements do we provide customers the ability to submit this feedback? Or is this for the MSFT engineers taking feedback and providing it to the PGs?

Reply:
More of the latter I believe, but for customers' visibility as well when wording their feedback to be relayed by our engineers.
On the Windows container questions not sure this is the right place, it was a P0 feedback with multiple customers attached, it's easy to find it in our Feedback tool for more info.

Reply:
Updated the PR title, hopefully it's a little more clear